reporter/src/main/kotlin/model/EvaluatedModel.kt

```diff
@@ -19,7 +19,284 @@
 
 package com.here.ort.reporter.model
 
+import com.fasterxml.jackson.annotation.JsonIdentityInfo
```

Comment:
I'd vote for having separate files per class.

Reply:
You mean for each class in this file? I fine with that but would prefer to do that change in a separate PR.